src/client/components/JudgePage.tsx

```diff
@@ -41,21 +42,27 @@ const JudgeSubPage = (props: JudgePageProps) => {
       return (<JudgePageEnded />);
     case "JUDGE_SUBPAGE_ASSIGNMENT_RATING":
       return (
-        <JudgePageAssignmentRating
-          assignmentId={props.assignmentId!}
-          judgeName={props.judgeName!}
-          ratingAssignment={props.ratingAssignment!}
-          onSubmitRatingAssignment={props.onSubmitRatingAssignment!}
-        />
+        <div>
+          <Timer start={Date.now()} />
+          <JudgePageAssignmentRating
+            assignmentId={props.assignmentId!}
+            judgeName={props.judgeName!}
+            ratingAssignment={props.ratingAssignment!}
+            onSubmitRatingAssignment={props.onSubmitRatingAssignment!}
+          />
+        </div>
       );
     case "JUDGE_SUBPAGE_ASSIGNMENT_RANKING":
       return (
-        <JudgePageAssignmentRanking
-          assignmentId={props.assignmentId!}
-          judgeName={props.judgeName!}
-          rankingAssignment={props.rankingAssignment!}
-          onSubmitRankingAssignment={props.onSubmitRankingAssignment!}
-        />
+        <div>
+          <Timer start={Date.now()} />
```

Comment:
Can you add a screenshot of what the timer looks like on the page. I would usually just run the code, but I don't have sledge setup on my laptop